page/performance/optimize-selectors.md

```diff
@@ -7,19 +7,28 @@
 
 Selector optimization is less important than it used to be, as more browsers implement `document.querySelectorAll()` and the burden of selection shifts from jQuery to the browser. However, there are still some tips to keep in mind.
 
-## ID-Based Selectors
+## Save calls to `querySelectorAll`
 
-Beginning your selector with an ID is always best.
+```
```

Comment:
I think this  example might need a small introduction.  Even 1-2 sentences should do it.